src/Neo.CLI/CLI/MainService.cs

```diff
@@ -366,26 +368,25 @@ public void OpenWallet(string path, string password)
             CurrentWallet = Wallet.Open(path, password, NeoSystem.Settings) ?? throw new NotSupportedException();
         }
 
-        public async void Start(string[] args)
+        public async void Start(CommandLineOptions options)
         {
             if (NeoSystem != null) return;
-            bool verifyImport = true;
-            for (int i = 0; i < args.Length; i++)
-                switch (args[i])
-                {
-                    case "/noverify":
-                    case "--noverify":
-                        verifyImport = false;
-                        break;
-                }
+            bool verifyImport = options.NoVerify ?? true;
```

Comment:
```suggestion
            bool verifyImport = !(options.NoVerify ?? false);
```

We equal NoVerify to verify, it must be the opposite, isn't it?